.buildscripts/deploy_snapshot.sh

```diff
@@ -9,6 +9,7 @@
 REPO="purchases-android"
 USERNAME="RevenueCat"
 JDK="oraclejdk8"
+VERSION=$(grep "versionName" purchases/build.gradle | awk '{print $2}' | sed -e 's/^"//' -e 's/"$//')
```

Comment:
the more complex this stuff gets, the more I feel like it might be worth the investment to move this to [fastlane](https://docs.fastlane.tools/actions/nexus_upload/), especially since it already has convenience methods for getting version numbers and stuff. And it's much easier to maintain ruby than sh